### DISCUSSION TITLE
Sponsoring on mobile!

### DISCUSSION THREAD

**Discussion by liaujianjie** [community/community#7446] [Other Feature Feedback, Questions, & Ideas]
It will be cool if we can sponsor projects from the explore tab.

**Comment by jlord** [comment]
Hi! We'd love to make sponsoring easier and in more places but it becomes tricky on the mobile apps because of fees required by the app stores that would change how much went to maintainers compared to the flow on the website.

**Reply by terrytangyuan** [reply]
I think even displaying sponsorships on mobile would be hugely helpful. Perhaps on mobile there would be a sponsor button that redirects to GitHub website for the actual payment flow.

**Comment by Deadlyelder** [comment]
How about having an option to setup something like `Remind me` on mobile about sponsoring? It would remind you in notification when you log in the web to finish the actual payment flow
